src/types.d.ts

```diff
@@ -159,6 +160,9 @@ export interface ScatterplotMethodOptions {
     transitionDuration: number;
     transitionEasing: (t: number) => number;
     preventFilterReset: boolean;
+    hover: number;
+    select: number | number[];
+    focus: number | number[];
```

Comment:
I think these 2 should always expect an array, it introduces additional complexity and potential vulnerabilities in the codebase. If it's `selectedPoints`, then it's clear for the users that they need to submit: `[0]` for example.

Reply:
Both methods already support this signature. The complexity overhead is neglectable